inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -202,6 +156,7 @@ void FrontEnd::detectNetworkBatch(
 
             _unbatchedOutputs.insert(ieData);
```

Comment:
Could you find any real cases when `origShape == newShape` is true? It looks redundant to me.